libs/gamify/users/src/lib/users.controller.ts

```diff
@@ -0,0 +1,7 @@
+import { Controller } from '@nestjs/common';
+import { UsersService } from './users.service';
+
+@Controller('users')
+export class UsersController {
+  constructor(private usersService: UsersService) {}
```

Comment:
Any thoughts on what you want to achieve here? Return all database users?

Reply:
Yes, this controller will be used in future features including returning all users and assigning a badge to a user.